app/templates/main/upload.html

```diff
@@ -18,9 +18,17 @@
 
                 {% if not tab_errors %}
                     <h1 class="govuk-heading-l">Get ready to submit your monitoring and evaluation data return</h1>
+
+                    {% set notificationHTML %}
+                      <p class="govuk-notification-banner__heading">
+                        You have {{ days_remaining }} days left to submit your {{ returns_period }} return.
+                      </p>
+                    {% endset %}
+
                     {{ govukNotificationBanner ({
-                        "text": "You have 10 days left to submit your April to September 2023 return."
+                        "html": notificationHTML
                     }) }}
+
```

Comment:
similar comment to above - what do we want to display if they're past the deadline? Sounds like this is a guaranteed scenario

Reply:
There is a now new ticket for this scenario